Sources/OrbitStorybook/Detail/Items/StorybookDialog.swift

```diff
@@ -72,6 +62,7 @@ struct StorybookDialogPreviews: PreviewProvider {
     static var previews: some View {
         OrbitPreviewWrapper {
             StorybookDialog.basic
+            StorybookDialog.normal
```

Comment:
`basic` **and** `normal`? 🤔

Reply:
This is using the existing naming (basic stands for the tab content and the normal is used within the component preview)